tests/FolderTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
```

Comment:
Have you (the team) already made a decision for `@copyright` tag? Maybe give it to FSFE so they can handle any law issues for you. Maybe on the developer team (in general) to express that people develop code not for themselves but for the project?